src/components/common/modal/folderNameInput/FolderNameInput.tsx

```diff
@@ -0,0 +1,21 @@
+import { ChangeEvent, useState } from 'react';
+import styles from './FolderNameInput.module.css';
+
+const FolderNameInput = ({ folderName = '' }) => {
+  const [value, setValue] = useState(folderName);
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) =>
+    setValue(e.target.value);
+  const active = value ? 'active' : '';
+
+  return (
+    <input
+      type="text"
+      placeholder="내용 입력"
+      value={value}
+      className={`${styles.input} ${active && styles.active}`}
```

Comment:
c: 다음처럼 Array 메서드를 사용해 클래스네임 여러개를 이어붙일 수도 있어요. classnames 라이브러리가 이렇게 구현되었습니다.

```suggestion
      className={
        [styles.input, active && styles.active]
          // 값이 있는 요소만 필터링 (active가 false면 styles.active가 필터링에서 제외됨)
          .filter(Boolean)
          // 공백으로 붙이기
          .join(' ')
      }
```